common/persistence/nosql/nosqlTaskStore.go

```diff
@@ -237,7 +236,7 @@ func (t *nosqlTaskStore) CreateTasks(
 			WorkflowID:   t.Execution.GetWorkflowID(),
 			RunID:        t.Execution.GetRunID(),
 			ScheduledID:  t.Data.ScheduleID,
-			CreatedTime:  now,
+			CreatedTime:  t.Data.CreatedTime,
```

Comment:
Is there no test for this field? how about separating a different PR and add the test assertion?

Reply:
I do, just fixed the test and I realized we need similar changes for SQL. 
so I will revert those changes and do it in another pr.